cluster/gce/gci/configure-helper.sh

```diff
@@ -1880,6 +1888,11 @@ function start-kube-scheduler {
   if [[ -n "${SCHEDULING_ALGORITHM_PROVIDER:-}"  ]]; then
     params+=" --algorithm-provider=${SCHEDULING_ALGORITHM_PROVIDER}"
   fi
+  if [[ -n "${SCHEDULER_POLICY_CONFIG:-}" ]]; then
+    create-kubescheduler-policy-config
+    params+=" --use-legacy-policy-config"
```

Comment:
Why don't we create a ConfigMap object instead?

Reply:
I mentioned this in the reviewer note. kube-scheduler requires the ConfigMap to be set before it starts but it's not easy to so in GKE.

Reply:
Oh, sorry that I missed that in the reviewer node.
The reason that I think using a ConfigMap is better is that we are thinking about deprecating the config file. If you set the flag (`--policy-configmap`) when starting the scheduler and the ConfigMap is not found, scheduler will refuse to start. I assume our cluster bring up (or kubelet) will retry to run the scheduler again and this will be repeated until the ConfigMap is there.

Reply:
Per discussion offline, requiring a ConfigMap to be present before the scheduler starts introduces a dependency on etcd/apiserver and whichever component that should post the ConfigMap. There is currently no guarantee on the creation ordering of various components during cluster creation, and it's not deal to let the scheduler keep restarting before the config is available in this case, so we decided to use the file for now.